testsuite/forge.env

```diff
@@ -0,0 +1,6 @@
+# This file is source-d when running Forge
```

Comment:
beautiful and clear!

lets be sure to document this so that people know they can use this nice new interface

NOTE: we need to prevent people from modifying and committing this

Reply:
May be .gitignore this file but force add it to this PR, so other's cannot commit it unintentionally in the future.

Reply:
I tried to do this earlier but couldn't figure out an elegant solution. The issue is that we want folks to maybe modify and commit this (for their own tests), but never land it.

In `run_forge.sh` it fails forge after all the tests run, if there's any env vars set here, so I think that should cover all those cases.